packages/react/src/components/Storage/FileUploader/FileUploader/types.ts

```diff
@@ -13,4 +14,5 @@ export interface FileUploaderProps {
   shouldAutoProceed?: boolean;
   showImages?: boolean;
   variation?: 'drop' | 'button';
+  displayText?: Partial<DisplayText>;
```

Comment:
It would be great DX if we updated the docs when making changes to the API.

Reply:
I was going to handle docs in a separate PR because I want to generate docs from the source. This is on a feature branch so won't affect prod